profiles/portal/etc/supervisor/conf.d/xmlrpc-delayed-daemon.conf

```diff
@@ -0,0 +1,8 @@
+[program:xmlrpc]
```

Comment:
Maybe we can take this opportunity to rename this services to kamrpc-delayed instead of xmlrpc (Filename and this header)
Also its better (if not required) if this header name is unique